src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx

```diff
@@ -53,17 +53,26 @@ For users on the [New Relic One user model](/docs/accounts/original-accounts-bil
 
-To optimally use our more advanced user management features, it's important to first understand the concept of the "access grant." An access grant gives a group of users access to a specific role on a specific account. For a New Relic [organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure) that has many accounts, groups typically require more than one access grant because users in a group usually need access to multiple accounts and roles. The diagram below explains the elements that make up an access grant.
+If your organization has [Pro or Enterprise edition](https://newrelic.com/pricing), you can create and configure access grants. An access grant gives a group of users access to a specific role on a specific account. 
 
-Note that if your organization is on Standard [edition](https://newrelic.com/pricing) and you want to assign a user to a [default group](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#groups) (**Admin** or **User**), you don't need to create an access grant: you would simply [add a user](#add-users) to that group and you're done. But for Pro and Enterprise edition, if you're trying to grant users access to a custom group, a custom role, or to other accounts, you must create an access grant.
+Here's a diagram showing how access grants work and how they relate to the broader organization:
 
 ![New Relic user management diagram](./images/user-management-diagram.png "user-management-diagram.png")
 
 <figcaption>
-  A diagram explaining how you can grant user groups access to roles and accounts. Note that this applies to users on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) (and not our original user model).
+  A diagram explaining the concept of how access grants give a user group access to a) a role on b) a specific account. (Note that this applies to users on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) (and not our original user model).)
 </figcaption>
 
-For examples of some common user management tasks, see [Example tasks](#workflow). For a glossary of user management terms, see [Definitions](#definitions).
+When your New Relic organization is created, it starts out with some default access grants for our default [**User** or **Admin** groups](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups). Each of these groups is assigned one or more roles and granted access to the primary (first created) account: 
+
+![New Relic organization and access UI - default access grants](.images/organization-and-access-ui-default-groups.png "New Relic organization and access UI - default access grants")
```

Comment:
This image is broken.

Reply:
in my experience a lot of the image don't work for me in local builds but are actually ok. could it be that?

Reply:
huh it does seem broken but I can't for life of me see why it would be

Reply:
Let me see if I can figure it out...

Reply:
I found it. Pushing up a fix. In my experience, if an image is broken in a local build, it's broken everywhere.

Reply:
You were missing a forward slash between the period and "images"